src/Microsoft.DotNet.Wpf/src/PresentationCore/otherassemblyattrs.cs

```diff
@@ -21,6 +21,7 @@
 
 [assembly: TypeForwardedTo(typeof(System.Windows.Markup.IUriContext))]
 [assembly: TypeForwardedTo(typeof(System.Windows.Media.TextFormattingMode))]
+[assembly: TypeForwardedTo(typeof(System.Windows.Input.ICommand))]
```

Comment:
Was this missed previously?

Reply:
It was, originally, inside ICommand.cs.  I removed that file and bumped the forward to the attributes file so it could be easily included in the ref assembly.